tests/test_all_rules.py

```diff
@@ -80,7 +80,14 @@ def test_duplicate_file_names(self):
 
         duplicates = {name: paths for name, paths in name_map.items() if len(paths) > 1}
         if duplicates:
-            self.fail(f"Found duplicated file names {duplicates}")
+            self.fail(f"Found duplicated file names: {duplicates}")
+
+    def test_duplicate_rule_names(self):
```

Comment:
can you assert this in the rule loader instead

https://github.com/elastic/detection-rules/blob/ca0e4ac72ae965966f928adc046b692c763b595c/detection_rules/rule_loader.py#L207-L222

Reply:
Completed